marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/mappingUILib.sjs

```diff
@@ -0,0 +1,126 @@
+const entityLib = require("/data-hub/5/impl/entity-lib.sjs");
```

Comment:
Let's hold off on creating a lib here; there's no use case for it until we have multiple clients for this library. Keeping all the code in the DS endpoint is fine for now, particularly since it's specific to the one use case supported by that endpoint.

Reply:
The only reason I created a lib was to test individual functions. If not, I can only test the entire response.

Reply:
That's fair - how about mappable-entity-lib? It seems that "mappable entity" is the main concept here, where that term implies an entity model plus some mapping-specific info.

Reply:
Ok, I will rename the library